packages/next/client/index.tsx

```diff
@@ -42,7 +42,7 @@ type RenderRouteInfo = PrivateRouteInfo & { App: AppComponent }
 type RenderErrorProps = Omit<RenderRouteInfo, 'Component' | 'styleSheets'>
 
 if (!('finally' in Promise.prototype)) {
```

Comment:
isn't this conditional also already part of the polyfill?

Reply:
sure. And  I tried to run `yarn dev`, but there is no `finally-polyfill` in the `next/dist/build/polyfills/`. 
I didn't find the problem, but it should have nothing to do with my code.
![image](https://user-images.githubusercontent.com/15967520/91442245-1f8f7f80-e8a4-11ea-959d-2ade17b81eaf.png)